web/pandas/pdeps/0004-consistent-to-datetime-parsing.md

```diff
@@ -64,6 +64,11 @@ Out[3]:
 1   2000-01-13
 dtype: datetime64[ns]
 ```
+or, if their dates are all ISO8601,
```

Comment:
You can update the one above to use `format="mixed"` ?

Reply:
I missed this one - thanks!